numpy/lib/npyio.py

```diff
@@ -1572,6 +1572,9 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
         column, individually.
     comments : str, optional
         The character used to indicate the start of a comment.
+        By default, the character '#' starts a comment. If some other
+        character is used, override `deletechars` to preserve
+        special characters in the header names.
```

Comment:
Could you format these lines so that they all have about the same length? I don't quite understand the use of `deletechars` here.

Reply:
Refer [#13088(comment)](https://github.com/numpy/numpy/issues/13088#issuecomment-482915623)